src/bp-messages/classes/class-bp-rest-group-messages-endpoint.php

```diff
@@ -154,15 +154,24 @@ public function create_item( $request ) {
 
 				if ( '' !== $group_thread ) {
 					// phpcs:ignore
-					$total_threads = $wpdb->get_results( $wpdb->prepare( "SELECT is_deleted FROM {$bp->messages->table_name_recipients} WHERE thread_id = %d", (int) $group_thread ) ); // db call ok; no-cache ok;
-					foreach ( $total_threads as $thread ) {
-						if ( 1 === (int) $thread->is_deleted ) {
-							$is_deleted = true;
-							break;
+					//$total_threads = $wpdb->get_results( $wpdb->prepare( "SELECT is_deleted FROM {$bp->messages->table_name_recipients} WHERE thread_id = %d", (int) $group_thread ) ); // db call ok; no-cache ok;
+					$total_threads = BP_Messages_Thread::get(
+						array(
```

Comment:
@janakkaneriya007 I don't think we need to pass `field` param as we returning full object